kafka-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/kafka/autoconfig/KafkaAutoConfiguration.java

```diff
@@ -152,6 +152,8 @@ private boolean isNonEmptyString(String s) {
                                          clazz -> clazz.isAssignableFrom(KafkaEventPublisher.class)
                                  );
 
+        eventProcessingConfigurer.assignHandlerInstancesMatching(DEFAULT_PROCESSING_GROUP, kafkaEventPublisher::equals);
```

Comment:
Nit: As stated on the original issue, this addition is a necessity because the `EventProcessingConfigurer` contains a bug which will only be resolved in Axon Framework version 4.4.3, whereas this extension is based on 4.0.4. 

I'd like you to add a comment here to specify this, just so we know we can remove this line as soon as the version of this extension is also based on 4.4.x or higher.